src/collective/nitf/profiles/default/registry.xml

```diff
@@ -77,4 +77,10 @@
       <value key="group" i18n:translate="">NITF</value>
   </records>
 
+  <record name="plone.app.tiles">
```

Comment:
@hvelarde is it possible to add any condition here if collective.nitf is installed?